Tests/HummingbirdTests/FileMiddlewareTests.swift

```diff
@@ -316,6 +316,55 @@ final class FileMiddlewareTests: XCTestCase {
         }
     }
 
+    func testOnThrow404() async throws {
```

Comment:
Not sure this test is required as all the other FileMiddleware tests are relying on this. 

The second test of course is necessary.